cpp/include/Ice/OutgoingAsync.h

```diff
@@ -145,7 +145,6 @@ namespace IceInternal
         virtual AsyncStatus invokeCollocated(CollocatedRequestHandler*) = 0;
 
         virtual bool exception(std::exception_ptr);
-        virtual void cancelable(const CancellationHandlerPtr&);
```

Comment:
It would be nice to cleanup this code and use override / final instead of virtual, where appropriate.

Here "virtual" meant "override".